judge/testcase_run.sh

```diff
@@ -212,6 +212,12 @@ runcheck "$RUN_SCRIPT" $RUNARGS \
 	--stderr=program.err --outmeta=program.meta -- \
 	"$PREFIX/$PROGRAM" 2>runguard.err
 
+if [ "$CREATE_WRITABLE_TEMP_DIR" ]; then
+	# Revoke access to the TMPDIR as security measure
+	chown -R "$(id -un):" "$TMPDIR"
+	chmod -R o-rwx "$TMPDIR"
```

Comment:
Nitpick: I'd write
```suggestion
	chmod -R go= "$TMPDIR"
```
to make it clear that there should be no permissions left (also not for group to be sure), including no sticky bit or other stuff. See also https://github.com/DOMjudge/domjudge/pull/2550/files#diff-5d7612b63b2d5e7da8f4277055da1bb1509ddf947bb1f797016d5a448bb8d909L42